projects/elife.yaml

```diff
@@ -738,6 +738,9 @@ generic-cdn:
                 articles:
                     hostname: "{instance}-elife-published.s3.amazonaws.com"
                     condition: req.url ~ "^/articles/"
+                digest:
```

Comment:
to be consistent with `articles` this should be plural, `digests`?

Reply:
not sure yet what the `surrogate-keys` would be without knowing the naming pattern, they can be added later. The default would be that they are cached for 24 hours however.